### ISSUE TITLE
Forcing photometry at known/true SN/SLQ positions

### ISSUE THREAD

**Issue by drphilmarshall** [DarkEnergyScienceCollaboration/Twinkles#298]
@wmwv Delighted to read [your post at community.lsst.org](https://community.lsst.org/t/how-do-i-do-forced-photometry-on-a-set-of-ra-dec/1074) about forced photometry at user-supplied sky positions! 

I think this is something we'd like to do as part of the Twinkles light curve analysis, to emulate a perfect deblender and focus on the light curve quality in that idealized scenario. Are you thinking along these lines? Tell us more about the question(s) you are interested in!

I guess doing this for Twinkles would involve an upgrade to the DM Level 2 recipe - are you working on this?

**Comment by wmwv** [comment]
Yes, I am working on exactly this.  I trust you'll note that the answers to the post on community.lsst.org  mentioned that there was no DM code to do this easily :)

I don't think it will be that bad, just a bit annoying.

**Comment by drphilmarshall** [comment]
OK sounds good. In our meeting we brought up making a CoaddSource lookalike
catalog and ingesting it into the pserv database, and wondering which code
package shoudl take care of that (possibly just the Twinkles pipeline,
rather than the Monitor).

**Comment by drphilmarshall** [comment]
@jbkalmbach I'm pretty sure this is something we'll want to do at some point, to emulate a magical SL SuperFit that has figured out where the lensed quasar image positions are, and does accurate forced photometry. Unless the astrometry is off, of course, in which case we'll be emulating a very bad SuperFit... Let's take a look at this with @wmwv at CMU next week.
